.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: php
 
 php:
-  - 7.1
+  - 7.4
```

Comment:
Could you please update `.platform.yml` to match this? It's 7.3 right now.

Reply:
Alternatively, roll back to 7.3 as your travis failures are 7.4-specific.

The issues:
 * phpunit-mock-objects never added 7.4 support. You'd have to bump the test suite to PHPUnit 8 or 9.
 * php_codesniffer should be at least ^3. I'm not sure why it's stuck on 32....?

Reply:
As a project, we have a composer.lock in this repo - probably need to update codesniffer

Reply:
yeah but the PR seemed to have a `composer update` committed

Reply:
> yeah but the PR seemed to have a composer update committed

Well it is a :
```
composer remove sami/sami
composer require --dev code-lts/doctum
```

But indeed quite a lot of packages where updated
I updated to use PHP 7.3